minifi-commons/minifi-commons-schema/src/main/java/org/apache/nifi/minifi/commons/schema/ProvenanceRepositorySchema.java

```diff
@@ -25,27 +25,38 @@
 import static org.apache.nifi.minifi.commons.schema.common.CommonPropertyKeys.PROVENANCE_REPO_KEY;
 
 public class ProvenanceRepositorySchema extends BaseSchema implements WritableSchema {
-    public static final String PROVENANCE_REPO_ROLLOVER_TIME_KEY = "provenance rollover time";
 
+    public static final String PROVENANCE_REPO_ROLLOVER_TIME_KEY = "provenance rollover time";
     public static final String DEFAULT_PROVENANCE_ROLLOVER_TIME = "1 min";
 
+    public static final String PROVENANCE_REPOSITORY_KEY = "repository";
```

Comment:
maybe implementation would be a better term as it is the provenance repository regardless?

Reply:
I initially had it as `repository.implementation` but decided that might be unnecessarily long. Simply `implementation` does seem a better fit.